src/firebaseConfig.js

```diff
@@ -12,5 +12,6 @@ const firebaseConfig = {
 // Initialize Firebase
 firebase.initializeApp(firebaseConfig);
 
-export default firebase.firestore();
-export const Auth = firebase.auth;
+export const fireStore = firebase.firestore();
```

Comment:
maybe you need import one of these constants as default. The tests aren't passing

Reply:
revised